hooks/module.go

```diff
@@ -0,0 +1,6 @@
+package hooks
+
+type Module interface {
+	Name() string
+	Hooks() []interface{}
```

Comment:
The Hooks() function is not necessarily if we represent each hook as an interface. This was my intention from our discussions. For example, the bid request hook interface can be defined as: something like

```
type BidRequestHook interface {
  HandleBidRequest(...)
}
```

The module passively supports the hook if it implements that interface.